Library/Homebrew/download_strategy.rb

```diff
@@ -528,6 +528,8 @@ def _fetch(url:, resolved_url:, timeout:)
   def _curl_args
     args = []
 
+    args += ["-L"] if Homebrew::EnvConfig.artifact_domain
```

Comment:
Are you sure this isn't already passed? If not: what's the error (with `--debug`)? Would like to add this more specific to the location where it's needed.

Reply:
You are right. This is not neccessary.